app/src/main/java/com/spendster/presentation/addExpenses/chooseCategory/OnItemClickListener.java

```diff
@@ -0,0 +1,7 @@
+package com.spendster.presentation.addExpenses.chooseCategory;
+
+import com.spendster.data.entity.Category;
+
+public interface OnItemClickListener {
```

Comment:
@SimonMaha naming is wrong. It works only with category adapter but is being named in a generic way.